bridge/gitlab/iterator.go

```diff
@@ -25,6 +26,18 @@ type labelEventIterator struct {
 	cache []*gitlab.LabelEvent
 }
 
+func (l *labelEventIterator) Len() int {
+	return len(l.cache)
+}
+
+func (l *labelEventIterator) Swap(i, j int) {
+	l.cache[i], l.cache[j] = l.cache[j], l.cache[i]
```

Comment:
I didn't know that this is possible in Go. Pretty interesting. 
https://stackoverflow.com/questions/35707222/swap-two-numbers-golang/35707958